$MirOS: ports/emulators/qemu/stable/patches/patch-target-sparc_cpu_h,v 1.1 2008/05/07 15:46:13 tg Exp $
--- target-sparc/cpu.h.orig	Sun Sep  4 17:11:09 2005
+++ target-sparc/cpu.h	Tue Nov 15 11:37:45 2005
@@ -284,8 +284,7 @@ void cpu_set_cwp(CPUSPARCState *env1, in
     } while (0)
 #endif
 
-struct siginfo;
-int cpu_sparc_signal_handler(int hostsignum, struct siginfo *info, void *puc);
+int cpu_sparc_signal_handler(int hostsignum, siginfo_t *info, void *puc);
 
 #include "cpu-all.h"
 
